HLTrigger/Configuration/python/HLT_75e33/modules/hltL3fL1TkSingleMu18Filtered20_cfi.py

```diff
@@ -0,0 +1,16 @@
+import FWCore.ParameterSet.Config as cms
+
+hltL3fL1TkSingleMu18Filtered20 = cms.EDFilter( "HLTMuonTrkL1TkMuFilter",
+    inputCandCollection = cms.InputTag("hltPhase2L3MuonCandidates"),
+    inputMuonCollection = cms.InputTag("hltPhase2L3Muons"),
+    maxAbsEta = cms.double(2.1),
+    maxNormalizedChi2 = cms.double(1e+99),
+    minMuonHits = cms.int32(-1),
+    minMuonStations = cms.int32(1),
+    minN = cms.uint32(1),
+    minPt = cms.double(20.0),
+    minTrkHits = cms.int32(-1),
+    l1GTAlgoBlockTag = cms.InputTag("l1tGTAlgoBlockProducer"),
+    l1GTAlgoNames = cms.vstring("pPuppiTauTkMuon42_18"),
+    saveTags = cms.bool(True)
+)
```

Comment:
Thanks for the comments, I quite don't understand the cosmetic changes supposed here. Are you suggesting to add a newline before the closing parentheses? After getting clarified, I'll fix the suggestion and squash all the commits. Thanks again for the review!

Reply:
the point is that you end several files without newlines, and gitHub shows that graphically with:

![Screenshot from 2024-09-13 08-39-41](https://github.com/user-attachments/assets/fe70d14e-717e-46e8-97c4-91122e47a07b)

It's an extremely minor point, but since we had a campaign to remove all of this (cf https://github.com/cms-sw/cmssw/pull/44239), I would prefer to not undo it.